locale/core.yml

```
@@ -410,19 +410,19 @@ core:
 
     # These translations are displayed as tooltips for discussion badges.
     badge:
-      hidden_tooltip: Hidden
+      hidden_tooltip: Schované
 
     # These translations are displayed as error messages.
     error:
       generic_message: "Jejda, něco se pokazilo. Obnov prosím stránku a zkus to znovu."
-      not_found_message: The requested resource was not found.
-      permission_denied_message: You do not have permission to do that.
-      rate_limit_exceeded_message: You're going a little too quickly. Please try again in a few seconds.
+      not_found_message: Požadovaný materiál nebyl nalezen
+      permission_denied_message: Na tuto operaci nemáte dostatečná práva.
+      rate_limit_exceeded_message: Jste moc rychlý, zpomalte
```

Ve zbytku překladu používáme tykání tak jsem to použil i tady. A taky je problém, že původní řetězec je v mužském rodě pokud nechci psát zvrácenosti jako `rychlý/á`.

```suggestion
      rate_limit_exceeded_message: Příliš spěcháš. Zkus to prosím znovu za pár sekund.
```